pinot-segment-local/src/main/java/org/apache/pinot/segment/local/utils/TableConfigUtils.java

```diff
@@ -223,7 +223,7 @@ public static void validateInstancePoolsNReplicaGroups(TableConfig tableConfig)
 
   private static Set<ValidationType> parseTypesToSkipString(@Nullable String typesToSkip) {
     return typesToSkip == null ? Collections.emptySet()
-        : Arrays.stream(typesToSkip.split(",")).map(s -> ValidationType.valueOf(s.toUpperCase()))
+            : Arrays.stream(typesToSkip.split(",")).map(s -> ValidationType.valueOf(s.toUpperCase()))
```

Comment:
let's use pinot checkstyle and revert unrelated format changes.

Reply:
thanks for addressing!